lib/generators/dynamo_db/session_store_migration/templates/session_store_migration.erb

```diff
@@ -2,10 +2,12 @@
 
 class <%= name.camelize %> < ActiveRecord::Migration[<%= migration_version %>]
   def up
-    Aws::SessionStore::DynamoDB::Table.create_table
+    options = Rails.application.config.session_options
+    Aws::SessionStore::DynamoDB::Table.create_table(options)
```

Comment:
Probably dumb question - does this need to be idempotent? (Ie check if table already exists/update?)

Reply:
I appreciate that this is idempotent so I can easily include in code which gets executed at every bin/dev and/or deploy.